Twig/Extension/TemplateExtension.php

```diff
@@ -28,19 +27,12 @@ class TemplateExtension extends \Twig_Extension
     protected $modelAdapter;
 
     /**
-     * @var TranslatorInterface
+     * @param bool             $debug        Is Symfony debug enabled?
+     * @param AdapterInterface $modelAdapter A Sonata model adapter
      */
-    protected $translator;
-
-    /**
-     * @param bool                $debug        Is Symfony debug enabled?
-     * @param TranslatorInterface $translator   Symfony Translator service
-     * @param AdapterInterface    $modelAdapter A Sonata model adapter
-     */
-    public function __construct($debug, TranslatorInterface $translator, AdapterInterface $modelAdapter)
+    public function __construct($debug, AdapterInterface $modelAdapter)
```

Comment:
What about this BC-break? Do we consider it ok?

Reply:
Oh wait this is for master, so ok.

Reply:
Don't know how to handle this without a big pain. So I created this for the master branch.